cmd/root.go

```diff
@@ -153,6 +154,11 @@ func setRootFlags(flags *pflag.FlagSet) {
 		"platform", "", "",
 		"an optional platform specifier for container image sources (e.g. 'linux/arm64', 'linux/arm64/v8', 'arm64', 'linux')",
 	)
+
+	flags.String(
+		"key", "",
```

Comment:
I think we want the default here to be "cosign.key" since that's the default that syft uses as well.
```suggestion
		"key", "cosign.key",
```